modules/src/core/ics26_routing/handler.rs

```diff
@@ -77,7 +79,21 @@ where
         }
 
         Ics4ChannelMsg(msg) => {
-            let handler_output = ics4_msg_dispatcher(ctx, msg).map_err(Error::ics04_channel)?;
+            let ctx_ro: &Ctx = ctx;
```

Comment:
Why is `ctx_ro` necessary? Can't we just pass `ctx` around directly?

Reply:
https://github.com/informalsystems/ibc-rs/pull/1838/commits/3b138bbeb6fba603e4efd66996bfcd189391cb2c